proving_system/stark/src/cairo_vm/cairo_instruction.rs

```diff
@@ -0,0 +1,545 @@
+/*
+   TO DO: Find a better name
```

Comment:
Oh, we should remove this, or change the name. I think we can add the rest of the decode and see how we want to organize this better. I'm thinking having two files may be useful, since the code will become longer